### DISCUSSION TITLE
Fabric Support

### DISCUSSION THREAD

**Discussion by TheCherry** [EwyBoy/OreTweaker#74] [Ideas]
Hi,

I looking for something like OreTweaker but for fabric 1.18.2.
Your World-Stripper mod already is available for fabric, so do you plan to do the same for OreTweaker?
Will be neat.

Greetings

**Comment by EwyBoy** [comment]
I been looking into doing a Fabric port but Ore Tweaker currently does a lot of technical magic I have currently only been able to get to work on the forge version. Once I have cracked the code on how to disable ores for Fabric it might be possible.

I will look into this topic further and moved it here to the discussion forum.

**Comment by TheCherry** [comment]
For me it will be enough in the "first" version - if only Custom-Ore-Generation is available.
The ores generation that i want tweak can I disable by the mod itself - then I can create with your mod some configs to create my own version of it. My goal is that the ores from that mod is not only in the overworld, I added some extra dimensions to my modpack and now I need a way to "force" the players to visit this dimensions.

**Reply by EwyBoy** [reply]
Gottcha, I can put something together once I have time.
